maintainers/maintainer-list.nix

```diff
@@ -6250,6 +6250,12 @@
     githubId = 15692230;
     name = "Muhammad Herdiansyah";
   };
+  koozz = {
```

Comment:
Please cherry-pick the commit into the PR where you are adding yourself as a maintainer to a package. This makes sure all maintainers entries are actually used and we do not collect stale ones.

Reply:
That was the wrong template. Sorry.

----

Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
I have split the maintainer and the python(lib+cli) into two commits.